modules/layers/src/solid-polygon-layer/polygon.js

```diff
@@ -107,14 +111,9 @@ function copyNestedRing(target, targetStartIndex, simplePolygon, size, windingDi
     }
   }
 
-  if (windingDirection) {
-    modifyPolygonWindingDirection(target, windingDirection, {
-      start: targetStartIndex,
-      end: targetIndex,
-      size,
-      isClosed: true
-    });
-  }
+  windingOptions.start = targetStartIndex;
+  windingOptions.end = targetIndex;
+  modifyPolygonWindingDirection(target, windingDirection, windingOptions);
```

Comment:
Would set windingOptions.size here as well as it is a bit confusing why the passed size arg is ignored. Not sure that there are any noticeable performance differences.